portfolio/src/main/webapp/index.html

```diff
@@ -56,12 +56,12 @@ <h2>ABOUT ME</h2>
       <h2>Get to know me more!</h2>
       <p>Click here to learn a random fact about me:</p>
       <!-- TODO:  Query to JS file -->
-        <button onclick="addRandomFact()">Jen Fact!</button>
-        <div id="fact-container"></div> 
+      <button onclick="addRandomFact()">Jen Fact!</button>
+     <div><p id="fact-container"></p></div> 
```

Comment:
Nit: Just my preference so feel free to ignore! Generally I believe it's easier to read when tags are separated by newline. Also make sure to align this div with the other tags (unless it's intentionally indented) I was a little confused as to which section the div was encompassing because of the indentation.